test.sh

```diff
@@ -3,11 +3,11 @@
 source env/bin/activate
 echo ""
 echo "checking for linting errors"
-flake8 --select E,W --max-line-length=140 --ignore E722 jarviscli/
+flake8 --select E,W --max-line-length=140 --ignore E722 jarviscli/ installer
 echo "lint errors checked"
 echo ""
 cd jarviscli/
 echo "checking for unit test"
 python -m unittest discover
```

Comment:
Make it python3, if anyone has 2 versions

Reply:
> Make it python3, if anyone has 2 versions

Actually - if run in virtualenv 'python' (and we don't allow python 2.7 virtualenvs any more) should always be python 3.

> I don't think we need options for python now on first screen.

You are right! Had all the code, but 


About the Linux-Error:

Of course - yes, thanks! Didn't check Python 2 compatibility any more, because that's I was removing... But makes sense that the installer still works with Python 2.

Windows-Error is just stupid ;0.